search/lib/cmr.js

```diff
@@ -251,28 +269,24 @@ async function convertParams (providerId, params = {}) {
     });
     logger.debug(`Params: ${JSON.stringify(params)}`);
     logger.debug(`Converted Params: ${JSON.stringify(converted)}`);
-    return fromEntries(converted);
+    return Object.assign({ provider: providerId }, fromEntries(converted));
```

Comment:
you got rid of a lot of object assigns, why does this one exist

Reply:
I think it's because for the most part the other Object assigns were replaced with calls to this function. Behavior was inconsistent, with some functions adding providerId themselves to the params.  This cleans things up I think.